### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by rutu14**
Category cards on click, redirect to register page currently.

On clicking avatar, logout menu appears.

### CODE REVIEW

**Review by DivyanshSareen — APPROVED** [review]
good to merge

File: css/home.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+}
+
+.hero-title{
+    height: 50rem;
+    word-spacing: 9999rem;
+    border-radius: 10px;
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
```
Comment (sudipkundu999) [inline css/home.css]: use shorthand

File: css/home.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
+.hero-section{
+    margin: 2rem 4rem 8rem;
+    height: 50rem;
+}
+
+.hero-img{
+    height: inherit;
+    border-radius: 10px;
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
```
Comment (sudipkundu999) [inline css/home.css]: use shorthand

**Review by sudipkundu999 — APPROVED** [review]
Good to merge with changes

File: css/home.css (hunk 1 above)
Comment (rutu14) [reply css/home.css]: Sure

File: css/home.css (hunk 2 above)
Comment (rutu14) [reply css/home.css]: Sure

**Pull request merged**